### PR TITLE
infra: use the pg check in the right place

### PR DESCRIPTION
**Pull request by leseb**
Use the pg check before doing the pg check, not on the quorum check.
Also never quote int when doing comparaison.

Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
jenkins test all

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-update_cluste

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-update_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-bluestore_dmcrypt_journal

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-bluestore_dmcrypt_journal_collocation

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-switch_to_containers

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-centos7_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-update_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-switch_to_containers

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-centos7_cluster

**Comment by leseb** [comment]
jenkins test all

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-bluestore_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-docker_dedicated_journal

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-update_dmcrypt

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-update_docker_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-centos7_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-purge_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-switch_to_containers

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-update_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-centos7_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.3-journal_collocation_auto_dmcrypt

**Comment by guits** [comment]
jenkins test all

File: roles/ceph-mgr/tasks/pre_requisite.yml [hunk 1]
```diff
@@ -1,4 +1,21 @@
 ---
+- name: install redhat ceph-mgr package
+  package:
+    name: ceph-mgr
+    state: "{{ (upgrade_ceph_packages|bool) | ternary('latest','present') }}"
+  when:
+    - mgr_group_name in group_names
```
Comment (leseb) [inline roles/ceph-mgr/tasks/pre_requisite.yml]: remove this?

File: roles/ceph-mgr/tasks/pre_requisite.yml [hunk 2]
```diff
@@ -1,4 +1,21 @@
...
+    - ansible_os_family == 'RedHat'
+
+- name: install ceph mgr for debian
+  package:
+    name: ceph-mgr
+    state: "{{ (upgrade_ceph_packages|bool) | ternary('latest','present') }}"
+    default_release: "{{ ceph_stable_release_uca | default(omit) }}{{ ansible_distribution_release ~ '-backports' if ceph_origin == 'distro' and ceph_use_distro_backports else ''}}"
+  when:
+    - mgr_group_name in group_names
```
Comment (leseb) [inline roles/ceph-mgr/tasks/pre_requisite.yml]: remove this?

File: roles/ceph-mgr/tasks/pre_requisite.yml [hunk 3]
```diff
@@ -1,4 +1,21 @@
...
+- name: install redhat ceph-mgr package
+  package:
+    name: ceph-mgr
+    state: "{{ (upgrade_ceph_packages|bool) | ternary('latest','present') }}"
+  when:
+    - mgr_group_name in group_names
+    - ansible_os_family == 'RedHat'
+
+- name: install ceph mgr for debian
+  package:
```
Comment (leseb) [inline roles/ceph-mgr/tasks/pre_requisite.yml]: apt module?

File: roles/ceph-mgr/tasks/pre_requisite.yml [hunk 4]
```diff
@@ -1,4 +1,21 @@
 ---
+- name: install redhat ceph-mgr package
+  package:
```
Comment (leseb) [inline roles/ceph-mgr/tasks/pre_requisite.yml]: yum module?

Comment (guits) [reply roles/ceph-mgr/tasks/pre_requisite.yml]: for this one we can keep `package`

**Comment by guits** [comment]
errors are unrelated to this PR, we need to merge this PR in order to stabilize the CI

**Pull request merged**